var/spack/repos/builtin/packages/libxc/package.py

```diff
@@ -30,8 +30,8 @@ class Libxc(AutotoolsPackage, CudaPackage):
     version("2.2.1", sha256="ade61c1fa4ed238edd56408fd8ee6c2e305a3d5753e160017e2a71817c98fd00")
 
     variant("shared", default=True, description="Build shared libraries")
-    variant("kxc",default=False, description="Build with third derivatives")
-    variant("lxc",default=False, description="Build with fourth derivatives")
+    variant("kxc", default=False, when="@5.0:", description="Build with third derivatives")
+    variant("lxc", default=False, when="@5.0:", description="Build with fourth derivatives")
```

Comment:
Minor suggestion.  (See line 37).

```suggestion
    variant("kxc", default=False, when="@5:", description="Build with third derivatives")
    variant("lxc", default=False, when="@5:", description="Build with fourth derivatives")
```